apple/swift-docc-render
# Use consistent aside styling for doc/article pages

Bug/issue #, if applicable: 89588748

## Summary

This updates the aside styling for tutorial-style article pages to match the same styling used in normal documentation pages. This does not impact the more minimal styling used for tutorial pages themselves.

Before:
<img width="1436" alt="Screenshot 2023-03-27 at 4 41 48 PM" src="https://user-images.githubusercontent.com/212918/228091893-fa153a17-96b8-4c5f-918a-7ffbc0cdc13c.png">

After:
<img width="1436" alt="Screenshot 2023-03-27 at 4 41 52 PM" src="https://user-images.githubusercontent.com/212918/228091915-0df11b42-27ed-4aac-8229-baacf14aed18.png">

## Testing

Steps:
1. Run `npm run build` to build this branch.
2. Configure `DOCC_HTML_DIR=/path/to/dist` when previewing example content with DocC.
3. Verify that documentation pages and tutorial article pages both use the same styling for asides.
4. Verify that tutorial pages still use the more minimal existing styling for asides.

[Example.docc.zip](https://github.com/apple/swift-docc-render/files/11083807/Example.docc.zip)

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
